lib/config.ts

```diff
@@ -1,5 +1,10 @@
 let GRAPHQL_HOSTNAME = process.env.NEXT_PUBLIC_GRAPHQL_HOSTNAME as string
 
+// we need an internal dns to properly propagate the ip related headers to api
+// if we use the api endpoints, nginx will rewrite the header to prevent spoofing
+let GRAPHQL_HOSTNAME_INTERNAL = process.env.NEXT_INTERNAL_GRAPHQL_HOSTNAME as string
```

Comment:
```suggestion
let GRAPHQL_HOSTNAME_INTERNAL = process.env.NEXT_PUBLIC_GRAPHQL_HOSTNAME_INTERNAL as string
```

Reply:
this is the internal kubernetes dns that we should have. so using the PUBLIC string here doesn't really make sense

Reply:
The `NEXT_PUBLIC` is a [special prefix](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) that needs to added to the names of env vars being exposed to the browser.